test/test_0310_conf_store.py

```diff
@@ -275,16 +275,16 @@ def test_310_202(self):
 
     def test_310_203(self):
         # test case: remove one md, keep another
-        dnsList1 = ["greenybtes2.de", "www.greenybtes2.de", "mail.greenybtes2.de"]
+        dnsList1 = ["greenbytes2.de", "www.greenbytes2.de", "mail.greenbytes2.de"]
```

Comment:
As a general rule, when I encounter pairs of similar tokens, I assume that the lesser used/lesser reasonable one is a typo. If this is an intentional thing, I'd suggest using something that clearly indicates that intention.